### PR TITLE
Accounts new

### PR DESCRIPTION
**Pull request by sasuke13**

### CODE REVIEW

File: src/accounts/serializers.py [hunk 1]
```diff
@@ -59,16 +60,17 @@ class UpdateUserPasswordDTOSerializer(serializers.Serializer):
...
+    username = serializers.CharField(max_length=32)
     linkedin_url = serializers.URLField()
     photo = serializers.ImageField()
     about = serializers.CharField()
+    short_about = serializers.CharField(max_length=50)
     rating = serializers.FloatField()
     cv_file = serializers.FileField()
     role_id = serializers.IntegerField()
-    # subscriped_projects = models.ManyToManyField(to=Project, related_name="subscription", on_delete=models.CASCADE)
+    subscribed_projects = serializers.PrimaryKeyRelatedField(queryset=Project.objects.all(), many=True)
```
Comment (OleksandrZhydyk) [inline src/accounts/serializers.py:73]: In future commits please refactor, this line, move this query to repository.

File: src/accounts/views.py
Comment (OleksandrZhydyk) [inline src/accounts/views.py:1]: :+1: next step will be creating logout view, where you should erase refresh_token from cookie

**Review by OleksandrZhydyk — APPROVED** [review]
Left the comments for the next sprint in the code

File: src/accounts/views.py
Comment (sasuke13) [reply src/accounts/views.py:1]: Got it, chief🫡

**Review by HyTacker20 — APPROVED** [review]

**Review by antony-kosenko — APPROVED** [review]
Good job!)

**Review by OlexiyVasylenkoDev — APPROVED** [review]

**Pull request merged**